saskatoon/harvest/forms.py

```diff
@@ -30,53 +36,43 @@ class RFPForm(forms.ModelForm[RFP]):
     class Meta:
         model = RFP
         fields = [
-            'first_name',
-            'last_name',
-            'email',
-            'phone',
-            'number_of_pickers',
-            'comment',
+            "first_name",
+            "last_name",
+            "email",
+            "phone",
+            "number_of_pickers",
+            "comment",
```

Comment:
Are these changes coming from yapf? Personally I'd like to keep single quotes for _keys_ and double quotes for string values, I find it helpful.